packages/public-docsite-setup/package.json

```diff
@@ -20,7 +20,7 @@
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
-    "lint": "just-scripts lint && eslint ./bin/*.js ./scripts/*.js"
```

Comment:
we need to revert this as lint task runs only against `src`  
<img width="1079" alt="image" src="https://github.com/user-attachments/assets/13ba0ea4-7eb4-468d-a68a-b572c7521a3b" />

wondering if the env variable set within the lint task would persist within shell of execution ( it should right ? )

Reply:
it would persist only within this lint task node process